survey/survey.css

```diff
@@ -0,0 +1,81 @@
+@import url('https://fonts.googleapis.com/css?family=Crimson+Pro&display=swap');
+
+body{
+    font-family: 'Crimson Pro';
+    font-size: 1.2em;
+    background-color: rgb(245, 218, 187);
+}
+.heading{
+    text-align: center;
+}
+.style{
+    list-style: none;
+    padding: 0px;
+    margin-top: 0px;
+}
+
+.form-wraper{
+    background-color: rgb(251, 252, 245);
+    text-align: center;
+    max-width: 800px;
+    margin: auto;
+    padding: 10px;
+    box-shadow: 0px 0px 8px 2px gray;
```

Comment:
just 0 instead of 0px

Reply:
0 does not work

Reply:
`box-shadow: 0 0 8px 2px gray;`
doesn't work?
its not something you have to do, its just a convention

Reply:
ahso
I thought  
`box-shadow: 0 0 8 2 gray;`
`box-shadow: 0 0 8px 2px gray;` works. Thanks